index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Holliday Planner</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="nav-bar">
+        <img src="./picture/Logo.png" alt="brand-logo" width="80px" height="auto">
+        <div class="nav-menu">
```

Comment:
avoid using to many html elements, try to simplify the nav bar, a div and the 3 links should be sufficient
+make sure that the html file is formatted/aligned correctly, your editor should do this on save